app/src/types/types.ts

```diff
@@ -11,3 +11,16 @@ export interface Member {
   walletAddress: string
   teamId: number
 }
+
+export interface MemberInput {
+  name: string
+  walletAddress: string
+  id?: string
+  isValid: boolean
+}
```

Comment:
i think we can seperate each type into single file, just my opinion

Reply:
and then add into `index.ts` to do barrel export https://medium.com/@klauskpm/do-a-barrel-export-aa5b79b76b05